tests/all-tests.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp.testSuite");
+
+require("../src/js/index.js");
+
+var jqUnit = fluid.require("jqUnit");
+var fs     = require("fs");
+
+gpii.test.mail.smtp.testSuite.mailOptions = {
+    from:    "sender@localhost",
+    to:      "recipient@localhost",
+    subject: "Test Subject",
+    text:    "Test Body"
+};
+
+gpii.test.mail.smtp.testSuite.isSaneResponse = function (error, info) {
+    jqUnit.assertNull("There should be no mail errors", error);
+
+    jqUnit.assertNotNull("There should message info returned...", info);
+    if (info) {
```

Comment:
Can eliminate this "if" block since you have verified that it is not null in the previous assertion - having all the assertions explode is acceptable behaviour for a failing test : P